### PR TITLE
Add genai.vector.encodeBatch to the list of procedures

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1412.surge.sh

Updated pages:
- [Procedures](https://neo4j-docs-operations-1412.surge.sh/operations-manual/5/reference/procedures)

File: modules/ROOT/pages/reference/procedures.adoc [hunk 1]
```diff
@@ -2034,6 +2040,22 @@ It is therefore not supported by the parallel runtime (introduced in Neo4j 5.13)
...
 ====
 
+[[procedure_genai_vector_encodeBatch]]
+.genai.vector.encodeBatch() label:aura-only[Available only on Aura]
+[cols="<15s,<85"]
+|===
+| Description
+a|
+Encode a given batch of resources as vectors using the named provider.
+For each element in the given resource LIST this returns:
```
Comment (nilsceberg) [inline modules/ROOT/pages/reference/procedures.adoc:2050]: Same questions here: link to GenAI docs? What do we do when releasing for on-prem?

**Review by nilsceberg — APPROVED** [review]

File: modules/ROOT/pages/reference/procedures.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/reference/procedures.adoc:2050]: We'll remove the label when we release on-prem and add a note in the Cypher manual.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1412.surge.sh

Updated pages:
- [Procedures](https://neo4j-docs-operations-1412.surge.sh/operations-manual/5/reference/procedures)

**Comment by renetapopova** [comment]
The failing procedure test is not related to the added procedure. Therefore, it is ok to merge this PR.

**Pull request merged**